packages/oscal-react-library/src/components/OSCALCatalog.tsx

```diff
@@ -81,6 +83,9 @@ export const OSCALCatalog: React.FC<OSCALCatalogProps> = ({
 
   return (
     <OSCALDocumentRoot>
+      <Stack justifyContent="flex-end" direction="row">
+        <OSCALParamsDialog params={catalog.params} />
+      </Stack>
```

Comment:
It'd rather not hide this behind a dialog here -- I would almost prefer to display the dialog contents directly. Can we get feedback from the team and @brian-easyd on this?

Reply:
Good point, it would be easy enough to switch that.

Reply:
With Dialog

![image](https://github.com/EasyDynamics/oscal-react-library/assets/39490074/b5d80c9d-860c-4e87-bb12-92144a296c2e)

---

Without Dialog

![image](https://github.com/EasyDynamics/oscal-react-library/assets/39490074/97a437ad-d28b-41a2-a253-70d9c9f82753)

Reply:
Can you show with 800-53?

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/EasyDynamics/oscal-react-library/assets/39490074/44858c7e-d27f-4273-a947-7bdc41b6f167)